tests/integration/policy/test_policy_pack/package-lock.json

```diff
@@ -0,0 +1,1564 @@
+{
+    "name": "aws-policy",
```

Comment:
This file should be deleted. It's leftover from an `npm install`, but we're running `yarn install` explicitly as part of the tests (and not checking-in the `yarn.lock` file).